Nick Cooley

CooleyNick opened the pull request:
Here is my finished M1 prework!

francepack commented on section4/exercises/person.rb:
```diff
@@ -1,5 +1,33 @@
+end
+
+  def change_info(n, a)
+    @name = n
```
Here is an example of a method that changes the value of a property.

francepack commented on section4/exercises/burrito.rb:
```diff
@@ -11,9 +11,21 @@ def initialize(protein, base, toppings)
     @toppings = toppings
   end
+  def add_topping
+    "lime"
```
These methods do not change the values of any properties. You will need assignments to do that. You can also use parameters so that these properties aren't changed to the same thing every time these methods are called.

francepack commented on final_prep/README.md:
```diff
@@ -32,24 +32,27 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: <script src="https://gist.github.com/CooleyNick/25143b8d481cd3cd7a215b437e5942e7.js"></script>
```
I was able to access these links, because the url is visible, but check how your markdown ended up [here](https://github.com/CooleyNick/backend_mod_1_prework/tree/main/final_prep) - review the syntax for links when using md. 

Then try going to those urls. I see code like blurbs. Check out your file extensions- right now, these are javascript files.

Make sure to fix these links and files for completion of prework.

francepack reviewed (commented):
@CooleyNick 
I am missing your mod0Hero work, as well as your code annotations. Perhaps your work didn't get pushed up, or please review the other files in this directory and complete and fulfill the requirements of prework.

Also please revise the links to other final prep work, pointed out in feedback below. 

You can also work on methods that use parameters- this will come up in your Mod0Hero work, as well as your burrito class. But the first 2 items are most important for completion.

Please make the above additions and message me when this is completed, ideally by the 28th. Be in touch if you have questions.

CooleyNick commented:
Mason,

Thanks for the feedback! Let me double check on the Mod0Hero/annotations
work as I 100% completed this task. I will address your other notes as
well, and have these back to you by 9/28.

Nick

On Sat, Sep 25, 2021 at 11:15 PM Mason France ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
>
> @CooleyNick <https://github.com/CooleyNick>
> I am missing your mod0Hero work, as well as your code annotations. Perhaps
> your work didn't get pushed up, or please review the other files in this
> directory and complete and fulfill the requirements of prework.
>
> Also please revise the links to other final prep work, pointed out in
> feedback below.
>
> You can also work on methods that use parameters- this will come up in
> your Mod0Hero work, as well as your burrito class. But the first 2 items
> are most important for completion.
>
> Please make the above additions and message me when this is completed,
> ideally by the 28th. Be in touch if you have questions.
> ------------------------------
>
> In final_prep/README.md
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r716140385>
> :
>
> > @@ -32,24 +32,27 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
>
>  When you are finished, add screenshots of your calendar so we can provide feedback if needed!
>
> -- `Add Week 1 Screenshot Here`
> -- `Add Week 2 Screenshot Here`
> -- `Add Week 3 Screenshot Here`
> +- <img width="1055" alt="Screen Shot 2021-09-11 at 2 38 24 PM" src="https://user-images.githubusercontent.com/39220746/132961125-da3c5444-01dd-4522-b849-b2e6eb00ede9.png">
> +
> +- <img width="1051" alt="Screen Shot 2021-09-11 at 2 38 12 PM" src="https://user-images.githubusercontent.com/39220746/132961150-4707ba46-e5d7-4b0b-b468-15708943245c.png">
>
> It looks like you have a great start here. I say 'start' because a
> calendar is always a work in progress. A couple pieces of feedback:
>
>    - I like how you've broken up your days with personal time. Also, I
>    appreciate plenty of time typically allotted on the weekends- sometimes you
>    may need that much, but may also think about breaking up some of the longer
>    study blocks.
>    - I recommend a devoted time to review your calendar from lat week,
>    and make adjustments to the upcoming week.
>
> ------------------------------
>
> In section4/exercises/person.rb
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r716141524>
> :
>
> > +
> +  def initialize(n, a)
> +    @name = n
> +    @age = a
> +  end
> +
> +  def type_code
> +    "#{name} types Hello World!"
> +  end
> +
> +def eats_food
> +  "Mmmm bacon nom nom nom"
> +end
> +
> +  def change_info(n, a)
> +    @name = n
>
> Here is an example of a method that changes the value of a property.
> ------------------------------
>
> In section4/exercises/burrito.rb
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r716141615>
> :
>
> > @@ -11,9 +11,21 @@ def initialize(protein, base, toppings)
>      @base     = base
>      @toppings = toppings
>    end
> +  def add_topping
> +    "lime"
>
> These methods do not change the values of any properties. You will need
> assignments to do that. You can also use parameters so that these
> properties aren't changed to the same thing every time these methods are
> called.
> ------------------------------
>
> In final_prep/README.md
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r716141634>
> :
>
> >
>  ### Mentorship Prep
>  Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
>  - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
> -  - [ ] Add link to your gist here:
> +  - [ ] Add link to your gist here: <script src="https://gist.github.com/CooleyNick/25143b8d481cd3cd7a215b437e5942e7.js"></script>
>
> I was able to access these links, because the url is visible, but check
> how your markdown ended up here
> <https://github.com/CooleyNick/backend_mod_1_prework/tree/main/final_prep>
> - review the syntax for links when using md.
>
> Then try going to those urls. I see code like blurbs. Check out your file
> extensions- right now, these are javascript files.
>
> Make sure to fix these links and files for completion of prework.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#pullrequestreview-763652011>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJLHMCUKNVCNO2J3K6JVZ7TUD2UAHANCNFSM5D3OKROA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

francepack commented:
@CooleyNick 
Thanks- if you know you completed the work, and have it on your local device, the first thing I'd check is that those files got added, committed, and pushed.
If you push them, your PR should automatically update with the additions.

CooleyNick commented:
I added them, committed them, and then received an error message for the
push. I’m trying to figure that out as we speak.

On Mon, Sep 27, 2021 at 1:21 PM Mason France ***@***.***>
wrote:

> @CooleyNick <https://github.com/CooleyNick>
> Thanks- if you know you completed the work, and have it on your local
> device, the first thing I'd check is that those files got added, committed,
> and pushed.
> If you push them, your PR should automatically update with the additions.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#issuecomment-928198708>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJLHMCRI5VEAE3BDSJSHUVLUEC72RANCNFSM5D3OKROA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

francepack commented:
You may need to pull before you push. This could have happened if you
updated the repo directly on GitHub for portions. Like, added something to
the repo on GitHub.com rather than adding locally and pushing it up

On Mon, Sep 27, 2021, 3:04 PM CooleyNick ***@***.***> wrote:

> I added them, committed them, and then received an error message for the
> push. I’m trying to figure that out as we speak.
>
> On Mon, Sep 27, 2021 at 1:21 PM Mason France ***@***.***>
> wrote:
>
> > @CooleyNick <https://github.com/CooleyNick>
> > Thanks- if you know you completed the work, and have it on your local
> > device, the first thing I'd check is that those files got added,
> committed,
> > and pushed.
> > If you push them, your PR should automatically update with the additions.
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/turingschool/backend_mod_1_prework/pull/120#issuecomment-928198708
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AJLHMCRI5VEAE3BDSJSHUVLUEC72RANCNFSM5D3OKROA
> >
> > .
> > Triage notifications on the go with GitHub Mobile for iOS
> > <
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> >
> > or Android
> > <
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> >.
> >
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#issuecomment-928278546>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKSM6AGZW67D23MBL4J7SHLUEDL6VANCNFSM5D3OKROA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

CooleyNick commented:
That’s exactly what it said. Pull and then push again. I’ll give that a
shot. Thanks for the help on this.

On Mon, Sep 27, 2021 at 3:38 PM Mason France ***@***.***>
wrote:

> You may need to pull before you push. This could have happened if you
> updated the repo directly on GitHub for portions. Like, added something to
> the repo on GitHub.com rather than adding locally and pushing it up
>
> On Mon, Sep 27, 2021, 3:04 PM CooleyNick ***@***.***> wrote:
>
> > I added them, committed them, and then received an error message for the
> > push. I’m trying to figure that out as we speak.
> >
> > On Mon, Sep 27, 2021 at 1:21 PM Mason France ***@***.***>
> > wrote:
> >
> > > @CooleyNick <https://github.com/CooleyNick>
> > > Thanks- if you know you completed the work, and have it on your local
> > > device, the first thing I'd check is that those files got added,
> > committed,
> > > and pushed.
> > > If you push them, your PR should automatically update with the
> additions.
> > >
> > > —
> > > You are receiving this because you were mentioned.
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/turingschool/backend_mod_1_prework/pull/120#issuecomment-928198708
> > >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AJLHMCRI5VEAE3BDSJSHUVLUEC72RANCNFSM5D3OKROA
> > >
> > > .
> > > Triage notifications on the go with GitHub Mobile for iOS
> > > <
> >
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> > >
> > > or Android
> > > <
> >
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> > >.
> > >
> > >
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/turingschool/backend_mod_1_prework/pull/120#issuecomment-928278546
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AKSM6AGZW67D23MBL4J7SHLUEDL6VANCNFSM5D3OKROA
> >
> > .
> > Triage notifications on the go with GitHub Mobile for iOS
> > <
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> >
> > or Android
> > <
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> >.
> >
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#issuecomment-928309077>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJLHMCSEZO6SLETA3X7NGOLUEDP4DANCNFSM5D3OKROA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

CooleyNick commented:
I went ahead and pulled/pushed one more time without an error message. Are
you able to see the update?



On Mon, Sep 27, 2021 at 3:46 PM Nick Cooley ***@***.***> wrote:

> That’s exactly what it said. Pull and then push again. I’ll give that a
> shot. Thanks for the help on this.
>
> On Mon, Sep 27, 2021 at 3:38 PM Mason France ***@***.***>
> wrote:
>
>> You may need to pull before you push. This could have happened if you
>> updated the repo directly on GitHub for portions. Like, added something to
>> the repo on GitHub.com rather than adding locally and pushing it up
>>
>> On Mon, Sep 27, 2021, 3:04 PM CooleyNick ***@***.***> wrote:
>>
>> > I added them, committed them, and then received an error message for the
>> > push. I’m trying to figure that out as we speak.
>> >
>> > On Mon, Sep 27, 2021 at 1:21 PM Mason France ***@***.***>
>> > wrote:
>> >
>> > > @CooleyNick <https://github.com/CooleyNick>
>> > > Thanks- if you know you completed the work, and have it on your local
>> > > device, the first thing I'd check is that those files got added,
>> > committed,
>> > > and pushed.
>> > > If you push them, your PR should automatically update with the
>> additions.
>> > >
>> > > —
>> > > You are receiving this because you were mentioned.
>> > > Reply to this email directly, view it on GitHub
>> > > <
>> >
>> https://github.com/turingschool/backend_mod_1_prework/pull/120#issuecomment-928198708
>> > >,
>> > > or unsubscribe
>> > > <
>> >
>> https://github.com/notifications/unsubscribe-auth/AJLHMCRI5VEAE3BDSJSHUVLUEC72RANCNFSM5D3OKROA
>> > >
>> > > .
>> > > Triage notifications on the go with GitHub Mobile for iOS
>> > > <
>> >
>> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
>> > >
>> > > or Android
>> > > <
>> >
>> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
>> > >.
>> > >
>> > >
>> >
>> > —
>> > You are receiving this because you commented.
>> > Reply to this email directly, view it on GitHub
>> > <
>> https://github.com/turingschool/backend_mod_1_prework/pull/120#issuecomment-928278546
>> >,
>> > or unsubscribe
>> > <
>> https://github.com/notifications/unsubscribe-auth/AKSM6AGZW67D23MBL4J7SHLUEDL6VANCNFSM5D3OKROA
>> >
>> > .
>> > Triage notifications on the go with GitHub Mobile for iOS
>> > <
>> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
>> >
>> > or Android
>> > <
>> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
>> >.
>> >
>> >
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/turingschool/backend_mod_1_prework/pull/120#issuecomment-928309077>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AJLHMCSEZO6SLETA3X7NGOLUEDP4DANCNFSM5D3OKROA>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>>
>

CooleyNick commented:
Also, the links for the mentorship and reflection exercises have been
updated as far as I can tell.

On Tue, Sep 28, 2021 at 8:58 PM Nick Cooley ***@***.***> wrote:

> I went ahead and pulled/pushed one more time without an error message. Are
> you able to see the update?
>
>
>
> On Mon, Sep 27, 2021 at 3:46 PM Nick Cooley ***@***.***> wrote:
>
>> That’s exactly what it said. Pull and then push again. I’ll give that a
>> shot. Thanks for the help on this.
>>
>> On Mon, Sep 27, 2021 at 3:38 PM Mason France ***@***.***>
>> wrote:
>>
>>> You may need to pull before you push. This could have happened if you
>>> updated the repo directly on GitHub for portions. Like, added something
>>> to
>>> the repo on GitHub.com rather than adding locally and pushing it up
>>>
>>> On Mon, Sep 27, 2021, 3:04 PM CooleyNick ***@***.***> wrote:
>>>
>>> > I added them, committed them, and then received an error message for
>>> the
>>> > push. I’m trying to figure that out as we speak.
>>> >
>>> > On Mon, Sep 27, 2021 at 1:21 PM Mason France ***@***.***>
>>> > wrote:
>>> >
>>> > > @CooleyNick <https://github.com/CooleyNick>
>>> > > Thanks- if you know you completed the work, and have it on your local
>>> > > device, the first thing I'd check is that those files got added,
>>> > committed,
>>> > > and pushed.
>>> > > If you push them, your PR should automatically update with the
>>> additions.
>>> > >
>>> > > —
>>> > > You are receiving this because you were mentioned.
>>> > > Reply to this email directly, view it on GitHub
>>> > > <
>>> >
>>> https://github.com/turingschool/backend_mod_1_prework/pull/120#issuecomment-928198708
>>> > >,
>>> > > or unsubscribe
>>> > > <
>>> >
>>> https://github.com/notifications/unsubscribe-auth/AJLHMCRI5VEAE3BDSJSHUVLUEC72RANCNFSM5D3OKROA
>>> > >
>>> > > .
>>> > > Triage notifications on the go with GitHub Mobile for iOS
>>> > > <
>>> >
>>> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
>>> > >
>>> > > or Android
>>> > > <
>>> >
>>> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
>>> > >.
>>> > >
>>> > >
>>> >
>>> > —
>>> > You are receiving this because you commented.
>>> > Reply to this email directly, view it on GitHub
>>> > <
>>> https://github.com/turingschool/backend_mod_1_prework/pull/120#issuecomment-928278546
>>> >,
>>> > or unsubscribe
>>> > <
>>> https://github.com/notifications/unsubscribe-auth/AKSM6AGZW67D23MBL4J7SHLUEDL6VANCNFSM5D3OKROA
>>> >
>>> > .
>>> > Triage notifications on the go with GitHub Mobile for iOS
>>> > <
>>> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
>>> >
>>> > or Android
>>> > <
>>> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
>>> >.
>>> >
>>> >
>>>
>>> —
>>> You are receiving this because you were mentioned.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/turingschool/backend_mod_1_prework/pull/120#issuecomment-928309077>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AJLHMCSEZO6SLETA3X7NGOLUEDP4DANCNFSM5D3OKROA>
>>> .
>>> Triage notifications on the go with GitHub Mobile for iOS
>>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>>> or Android
>>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>>
>>>
>>

francepack commented on final_prep/README.md:
```diff
@@ -32,24 +32,27 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 - [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+  - [ ] Add a link to your gist here: https://gist.github.com/CooleyNick/9df4a32f0af7751a4495c3c83465ed2a
```
It sounds like you value communication, which will serve you well in teams. Turing provides ample opportunity to practice multiple forms of communication, as well as giving and receiving feedback.

francepack commented on final_prep/README.md:
```diff
@@ -32,24 +32,27 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 - [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+  - [ ] Add a link to your gist here:https://gist.github.com/CooleyNick/3aa4d5b2558feb09353662fa3d55fbce
```
Nice job overall. I personally would find it harder to take notes on unlined paper- to me, it is easier to space out notes that way, or highlight topics. Keep experimenting with note taking, keeping in mind how much you actually go back and reference notes, and what you can do to make them easier to revisit.

francepack commented on final_prep/README.md:
```diff
@@ -32,24 +32,27 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: https://gist.github.com/CooleyNick/9cab2580d2c9025fb0d51371be86f59a"
```
I think this may be a link to the wrong gist. This is the one with a letter to your future mentor.

francepack commented on final_prep/annotations.rb:
```diff
@@ -3,41 +3,72 @@
     'basic_info' => demographics,
+    #assign value "clothes" to key "clothes"
     'clothes' => clothes,
+    #assign key "exterior" to value "fur"
```
What are these values? Where do they come from? If a value is a variable, it can be useful to track where it comes from, like is it passed as a parameter, or was it declared previously.

francepack commented on final_prep/annotations.rb:
```diff
@@ -3,41 +3,72 @@
+    # same class as the interger 0
+    # varriable "i" into the parameter "num_2" is the
+    # same class as the interger 0
+    # The logical AND operator checks if both === methods returned TRUE
```
I would call these conditions, not methods. I also suggest doing some research on the `%` operator and discussing with classmates- it is a tricky one, but a great opportunity to practice technical explanation.

francepack commented on final_prep/mod_zero_hero.rb:
```diff
@@ -1,41 +1,65 @@
+puts sidekicks[0]
 # Print the last arch_enemy to the terminal
-
+puts arch_enemies[2]
```
Nice use of bracket notation. A next level to this task would be to come up with a way to print the last element in the array no matter how many elements are in there.

francepack commented on final_prep/mod_zero_hero.rb:
```diff
@@ -1,41 +1,65 @@
+save_the_day = "You can always count on Spiderman"
+bad_excuse = "Yikes! Not today!"
+
+if danger_level >= 50
```
I would expect to see this all wrapped up in a method. Right now, you call method assess_situation below, but I do not see that method definition.

francepack commented on final_prep/mod_zero_hero.rb:
```diff
@@ -1,41 +1,65 @@
+  puts bad_excuse
+elsif danger_level < 50
+  puts save_the_day
+elsif danger_level > 10
```
There should only be 1 elseif block that results in `save_the_day`. You can use the `&&` operator to make it so 2 conditions need to be satisfied.

francepack commented on final_prep/mod_zero_hero.rb:
```diff
@@ -1,41 +1,65 @@
+elsif danger_level > 10
+  puts save_the_day
+elseif danger_level < 10
+  puts "Meh. Hard pass."
```
All conditionals need an `else` block.

francepack commented on final_prep/mod_zero_hero.rb:
```diff
@@ -47,38 +71,72 @@
+class SuperHero
+  attr_accessor :name, :super_power, :age
+
+  arch_nemesis= "The Syntax Error"
```
You are missing a little syntax to make these attributes of your class. I recommend looking at your burrito class to find that.

francepack commented on final_prep/mod_zero_hero.rb:
```diff
@@ -47,38 +71,72 @@
+  end
+
+  def gain_power(num)
+    @num = num
```
This method does need a parameter, so good job there. But you need a reassignment of one of your attributes. Think about how to assign that attribute to itself plus the `num` passed through as a parameter. Right now, there is no reassignment.

francepack reviewed (commented):
@CooleyNick 
I do see your work now- thanks for checking on that!

Could you check one more thing? I think you may have just copied the wrong gist link for the first one in finalPrep. I have commented on the particular one below. 

Overall, I see the effort and completion, but there are a few things to look into during the time that leads up to Turing:
- Practice defining methods that use parameters, and calling them. I suggest using repl to set up your own code practice. 
- When you are doing this, try some conditionals where you pass in different info and hit different conditions.
- I also think experimenting with ruby prototype methods is good to try as it helps you see the kinds of processes that happen frequently in code.

Please let me know if you can get that one link updated, and also if you have questions. Keep pushing!

CooleyNick commented:
Thank you, Mason. After work, I will tackle this last piece. If there are
any other specific skills I should practice prior to starting, please let
me know. I’m getting excited.

On Wed, Sep 29, 2021 at 8:49 AM Mason France ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
>
> @CooleyNick <https://github.com/CooleyNick>
> I do see your work now- thanks for checking on that!
>
> Could you check one more thing? I think you may have just copied the wrong
> gist link for the first one in finalPrep. I have commented on the
> particular one below.
>
> Overall, I see the effort and completion, but there are a few things to
> look into during the time that leads up to Turing:
>
>    - Practice defining methods that use parameters, and calling them. I
>    suggest using repl to set up your own code practice.
>    - When you are doing this, try some conditionals where you pass in
>    different info and hit different conditions.
>    - I also think experimenting with ruby prototype methods is good to
>    try as it helps you see the kinds of processes that happen frequently in
>    code.
>
> Please let me know if you can get that one link updated, and also if you
> have questions. Keep pushing!
> ------------------------------
>
> In final_prep/README.md
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r718560083>
> :
>
> >
>  ### Group Work Prep
>  As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
>  - [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
> -  - [ ] Add a link to your gist here:
> +  - [ ] Add a link to your gist here: https://gist.github.com/CooleyNick/9df4a32f0af7751a4495c3c83465ed2a
>
> It sounds like you value communication, which will serve you well in
> teams. Turing provides ample opportunity to practice multiple forms of
> communication, as well as giving and receiving feedback.
> ------------------------------
>
> In final_prep/README.md
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r718562205>
> :
>
> >
>  ### Lesson Prep
>  You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
>  - [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
> -  - [ ] Add a link to your gist here:
> +  - [ ] Add a link to your gist here:https://gist.github.com/CooleyNick/3aa4d5b2558feb09353662fa3d55fbce
>
> Nice job overall. I personally would find it harder to take notes on
> unlined paper- to me, it is easier to space out notes that way, or
> highlight topics. Keep experimenting with note taking, keeping in mind how
> much you actually go back and reference notes, and what you can do to make
> them easier to revisit.
> ------------------------------
>
> In final_prep/README.md
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r718563380>
> :
>
> >
>  ### Mentorship Prep
>  Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
>  - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
> -  - [ ] Add link to your gist here:
> +  - [ ] Add link to your gist here: https://gist.github.com/CooleyNick/9cab2580d2c9025fb0d51371be86f59a"
>
> I think this may be a link to the wrong gist. This is the one with a
> letter to your future mentor.
> ------------------------------
>
> In final_prep/annotations.rb
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r718565036>
> :
>
> >      'clothes' => clothes,
> +    #assign key "exterior" to value "fur"
>
> What are these values? Where do they come from? If a value is a variable,
> it can be useful to track where it comes from, like is it passed as a
> parameter, or was it declared previously.
> ------------------------------
>
> In final_prep/annotations.rb
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r718566732>
> :
>
> >    (1..range).each do |i|
> +    # varriable "i" against parameter "num_1" is the
> +    # same class as the interger 0
> +    # varriable "i" into the parameter "num_2" is the
> +    # same class as the interger 0
> +    # The logical AND operator checks if both === methods returned TRUE
>
> I would call these conditions, not methods. I also suggest doing some
> research on the % operator and discussing with classmates- it is a tricky
> one, but a great opportunity to practice technical explanation.
> ------------------------------
>
> In final_prep/mod_zero_hero.rb
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r718567746>
> :
>
> >  # Print the last arch_enemy to the terminal
> -
> +puts arch_enemies[2]
>
> Nice use of bracket notation. A next level to this task would be to come
> up with a way to print the last element in the array no matter how many
> elements are in there.
> ------------------------------
>
> In final_prep/mod_zero_hero.rb
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r718570263>
> :
>
> >  #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
> -
> +danger_level = 52
> +save_the_day = "You can always count on Spiderman"
> +bad_excuse = "Yikes! Not today!"
> +
> +if danger_level >= 50
>
> I would expect to see this all wrapped up in a method. Right now, you call
> method assess_situation below, but I do not see that method definition.
> ------------------------------
>
> In final_prep/mod_zero_hero.rb
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r718572079>
> :
>
> >  #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
> -
> +danger_level = 52
> +save_the_day = "You can always count on Spiderman"
> +bad_excuse = "Yikes! Not today!"
> +
> +if danger_level >= 50
> +  puts bad_excuse
> +elsif danger_level < 50
> +  puts save_the_day
> +elsif danger_level > 10
>
> There should only be 1 elseif block that results in save_the_day. You can
> use the && operator to make it so 2 conditions need to be satisfied.
> ------------------------------
>
> In final_prep/mod_zero_hero.rb
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r718573512>
> :
>
> >  #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
> -
> +danger_level = 52
> +save_the_day = "You can always count on Spiderman"
> +bad_excuse = "Yikes! Not today!"
> +
> +if danger_level >= 50
> +  puts bad_excuse
> +elsif danger_level < 50
> +  puts save_the_day
> +elsif danger_level > 10
> +  puts save_the_day
> +elseif danger_level < 10
> +  puts "Meh. Hard pass."
>
> All conditionals need an else block.
> ------------------------------
>
> In final_prep/mod_zero_hero.rb
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r718576056>
> :
>
> >  #   - super_power
> -#   - age
> +#   - age
> +class SuperHero
> +  attr_accessor :name, :super_power, :age
> +
> +  arch_nemesis= "The Syntax Error"
>
> You are missing a little syntax to make these attributes of your class. I
> recommend looking at your burrito class to find that.
> ------------------------------
>
> In final_prep/mod_zero_hero.rb
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#discussion_r718578882>
> :
>
> > +  attr_accessor :name, :super_power, :age
> +
> +  arch_nemesis= "The Syntax Error"
> +  power_level = 100
> +  energy_level = 50
> +
> +  def say_name
> +    puts "#{name} Will save us!"
> +  end
> +
> +  def maximize_energy
> +    energy_level = 1000
> +  end
> +
> +  def gain_power(num)
> +    @num = num
>
> This method does need a parameter, so good job there. But you need a
> reassignment of one of your attributes. Think about how to assign that
> attribute to itself plus the num passed through as a parameter. Right
> now, there is no reassignment.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/turingschool/backend_mod_1_prework/pull/120#pullrequestreview-766749060>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJLHMCXOSTTLCXH5HPEH4CLUEMRPLANCNFSM5D3OKROA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
